airbyte-cdk/java/airbyte-cdk/s3-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/s3/S3ConsumerFactory.kt

```diff
@@ -151,6 +161,39 @@ class S3ConsumerFactory {
                 }
                 LOGGER.info { "Cleaning up destination completed." }
             }
+
+            // On stream success clean up the objects marked for deletion per stream. This is done
+            // on per-stream basis
+            streamSyncSummaries.forEach { entry: Map.Entry<StreamDescriptor, StreamSyncSummary> ->
```

Comment:
nit: I think you can do a destructuring assignment here?
```suggestion
            streamSyncSummaries.forEach { (streamDescriptor, syncSummary) ->
```

Reply:
done